release/ray_release/test_automation/state_machine.py

```diff
@@ -50,10 +72,48 @@ def _move_hook(self, from_state: TestState, to_state: TestState) -> None:
         Action performed when test transitions to a different state. This is where we do
         things like creating and closing github issues, trigger bisects, etc.
         """
-        if from_state == TestState.PASSING and to_state == TestState.FAILING:
+        if (
+            from_state == TestState.PASSING
+            and to_state == TestState.CONSITENTLY_FAILING
+        ):
+            self._create_github_issue()
+        elif (
+            from_state == TestState.FAILING
+            and to_state == TestState.CONSITENTLY_FAILING
+        ):
             self._create_github_issue()
-        elif from_state == TestState.FAILING and to_state == TestState.PASSING:
+        elif (
+            from_state == TestState.CONSITENTLY_FAILING
+            and to_state == TestState.PASSING
+        ):
             self._close_github_issue()
+        elif from_state == TestState.PASSING and to_state == TestState.FAILING:
+            self._trigger_bisect()
+
+    def _trigger_bisect(self) -> None:
```

Comment:
is it like we are going to trigger bisect automatically?

Reply:
it does yes